Stream_support/include/CGAL/IO/OFF/File_scanner_OFF.h

```diff
@@ -78,7 +79,7 @@ class File_scanner_OFF
     else
     {
       skip_comment();
-      std::string line;
+      line.clear();
```

Comment:
I don't know if this PR makes it in the next release, so we should cherry pick this change in IO.